electricpy/fault.py

```diff
@@ -1790,33 +1868,38 @@ def indmacheigenvalues(Lr,Ls,Lm,Rr,Rs,wrf=0,freq=60):
                 The Second Eigenvalue
     """
     # Calculate Required Values
-    omega_e_base = 2*_np.pi*freq
+    omega_e_base = 2 * _np.pi * freq
     omega_rf = wrf
-    torque_s = Ls/(omega_e_base*Rs)
-    torque_r = Lr/(omega_e_base*Rr)
+    torque_s = Ls / (omega_e_base * Rs)
+    torque_r = Lr / (omega_e_base * Rr)
     alpha = torque_r / torque_s
-    phi = 1 - Lm**2/(Ls*Lr)
+    phi = 1 - Lm**2 / (Ls * Lr)
     omega_r = omega_e_base
     # Calculate k1
-    k1 = -1/(2*phi*torque_r)*(1+alpha)
-    k1 += 1j*(omega_r/2-omega_rf)
+    k1 = -1 / (2 * phi * torque_r) * (1 + alpha)
+    k1 += 1j * (omega_r / 2 - omega_rf)
     # Calculate k2
-    k2 = 1/(2*phi*torque_r)
-    k2 *= _np.sqrt((1+alpha)**2-4*phi*alpha-(omega_r*phi*torque_r)**2
-                 +2j*(alpha-1)*omega_r*phi*torque_r)
+    k2 = 1 / (2 * phi * torque_r)
+    k2 *= _np.sqrt(
```

Comment:
- [ ] let's see if we can clean this formula up a bit, it seems messy and all over the place with this visual breakdown (though, admittedly, it wasn't much better before)

Reply:
not gonna fight this one.... LGTM (Looks good to me)